layouts/partials/components/announcement.html

```diff
@@ -1,7 +1,10 @@
 <section class="announcement">
     <div class="header-announcement">
         <div class="banner">
-          <p>End-to-End Multicloud Solutions.&nbsp;&nbsp;&nbsp;<strong>Solving Together.™</strong>&nbsp;&nbsp;&nbsp;Learn more at <a href="https://www.rackspace.com/">Rackspace.com</a></p>
+          <p>ALERT we would like to inform you that our website will be migrated on June 8th, you will not have any impact while this is happening.
+             <br>
+             We apologize for any inconvenience this may cause.
+          </p>
```

Comment:
I took a piece from the Change Request comms to customer and tweaked to fit in this banner. Then I reviewed with Kim Headen from Change Management and Anita Narani who will be our change sponsor for this Change Request. So far Anita agrees with something like this below...

```suggestion
          <p><strong>Non-Service Disrupting Maintenance:</strong> On June 8, 2023, Between 09:00 CDT and 12:00 CDT docs.rackspace.com will be redirected to our new Website.
          </p>
```